### DISCUSSION TITLE
Socketcluster error

### DISCUSSION THREAD

**Discussion by zazizazi** [cables-gl/cables_docs#217] [Answers & Questions] [answered]
Hi, suddenly the socketcluster op says Error: error in socketcluster connection (SocketProtocolError) when sending.
This is happening with the demo patch too.
Any Ideas why?

**Comment by steam0r** [comment, marked as answer]
hey, do you have an example patch for me? this one seems to be running fine for me on dev and live: https://dev.cables.gl/edit/mecjP3
